### PR TITLE
Email verification transaction status polling

### PR DESCRIPTION
**Pull request by prastoin**

### CODE REVIEW

File: packages/client/e2e/_utils/page.ts [hunk 1]
```diff
@@ -216,10 +216,6 @@ export async function bypassVerifyEmailScreen({
 
     await page.request.get(`${SERVER_ENDPOINT}/test/bypass-email-confirmation`);
 
-    //FIXME TMP should be removed after email polling
-    await page.reload();
```
Comment (Devessier) [inline packages/client/e2e/_utils/page.ts:220]: **👏 praise:** ‏It seems to be a little deletion, but it is due to a massive improvement.

File: packages/client/machines/appMachine.ts [hunk 2]
```diff
@@ -291,119 +291,198 @@ export function createAppMachine({
...
+                                        onError: {
+                                            target: 'debouncing',
                                         },
-                                    }),
+                                    },
+                                },
+
+                                debouncing: {
+                                    after: {
+                                        500: {
```
Comment (Devessier) [inline packages/client/machines/appMachine.ts:438]: **🎯 suggestion:** ‏I think 500ms is waaaaay too short. 5s would be better. I think it's better to let users wait some seconds than making two requests per second in background, for an undetermined amount of time. The application could be made slow for users with a bad internet connection.

File: packages/client/machines/appMachine.ts [hunk 3]
```diff
@@ -291,119 +291,198 @@ export function createAppMachine({
...
+                                    on: {
+                                        SIGN_OUT: undefined,
+                                    },
                                 },
                             },
 
                             on: {
-                                SIGN_OUT: undefined,
+                                SIGN_OUT: {
+                                    target: '.performSignOut',
```
Comment (Devessier) [inline packages/client/machines/appMachine.ts:481]: **👏 praise:** ‏Using an internal transition is definitely a good idea! I makes `SIGN_OUT: undefined` useless.

**Review by Devessier — DISMISSED** [review]
Good job! I love your polling implementation.
I think we should make delay between two polling requests longer before merging.

File: packages/client/__tests__/Authentication.test.tsx [hunk 4]
```diff
@@ -1875,6 +1883,38 @@ const authenticationModel = createModel<TestingContext>(
...
+                            playlistsCounter: user.playlistsCounter,
+                            followersCounter: user.followersCounter,
+                            followingCounter: user.followingCounter,
+                            devicesCounter: user.devicesCounter,
+                            hasConfirmedEmail: true,
+                        }),
+                    );
+                }),
+            ),
+        );
```
[the remark this answers is not in the document]
Comment (prastoin) [reply packages/client/__tests__/Authentication.test.tsx:1915]: Ahhh yes indeed ! I was wondering on how I could be able to retrieve the user we're impersonating !
Indeed gonna make the modication !

**Review by Devessier — APPROVED** [review]
Nice job!
We can merge.

**Pull request merged**